docs/api/core/Raycaster.html

```diff
@@ -98,6 +98,11 @@
 		This value shouldn't be negative and should be larger than the near property. 
 		</div>
 
+		<h3>[property:boolean visibleOnly]</h3>
+		<div>
+		Defines whether the raycaster should intersect only visible objects. Default is false. 
```

Comment:
I know it is not backwards-compatible, but defaulting `visibleOnly` to `true` seems more natural to me. What do you think?